.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2022 Helmholtz Centre for Environmental Research (UFZ)
```

Comment:
This file needs to moved one level up, isn't it?

Reply:
Yes. This also explains the failed dependabot check. Moved it one level up. Thanks for the hint.